KokoDajMu/KokoDajMu_MAIN/Classes/WriteInFile.cs

```diff
@@ -78,50 +78,50 @@ public void SaveAlbumListInTextFile()
 
         public void SaveSongListInTextFile()
         {
-            TextWriter convertSongListInTextFile = new StreamWriter(@"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\SongList.txt");
+            TextWriter convertSongListInTextFile = new StreamWriter(Constants.PATH_TO_TEXT_FILES_TO_WRITE + "SongList.txt");
 
             using (convertSongListInTextFile)
             {
                 foreach (Song song in this.songsList)
                 {
-                    convertSongListInTextFile.WriteLine("<song><{0}>[{1}]</song>", song.Name, song.SongDurationFromTXTFile);
+                    convertSongListInTextFile.WriteLine("<song><{0}>[{1}]</song>", song.Name, song.SongDurationFromFile);
                 }
             }
         }
 
         public void SaveAllFilesInOne()
         {
-            StreamWriter allFilesWrittenHere = File.CreateText((@"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\KokoDajMuOverWritten.txt"));
+            StreamWriter textFileOverWritten = File.CreateText(Constants.PATH_TO_TEXT_FILES_TO_WRITE + "KokoDajMuOverWritten.txt");
```

Comment:
All text file names could be constants.(KokoDajMuOverWritten.txt/UserList.txt/etc.)